examples/azure-service-mesh/security/cert-manager-letsencrypt/readme.md

```diff
@@ -0,0 +1,269 @@
+  Normal  Generated  14m   cert-manager-certificates-key-manager      Stored new private key in temporary Secret resource "bookinfo-letsencrypt-certs-hpjst"
+  Normal  Requested  14m   cert-manager-certificates-request-manager  Created new CertificateRequest resource "bookinfo-letsencrypt-certs-1"
+```
+
+3. check certificaterequest resource.
+```shell
+kubectl describe certificaterequest bookinfo-letsencrypt-certs-1 -n aks-istio-ingress
+```
+```console
+Name:         bookinfo-letsencrypt-certs-1
+Namespace:    aks-istio-ingress
+Labels:       <none>
+Annotations:  cert-manager.io/certificate-name: bookinfo-letsencrypt-certs
+              cert-manager.io/certificate-revision: 1
+              cert-manager.io/private-key-secret-name: bookinfo-letsencrypt-certs-hpjst
+API Version:  cert-manager.io/v1
+Kind:         CertificateRequest
+...
+Events:
+  Type    Reason              Age    From                                                Message
+  ----    ------              ----   ----                                                -------
+  Normal  WaitingForApproval  6m57s  cert-manager-certificaterequests-issuer-selfsigned  Not signing CertificateRequest until it is Approved
+  Normal  WaitingForApproval  6m57s  cert-manager-certificaterequests-issuer-vault       Not signing CertificateRequest until it is Approved
+  Normal  WaitingForApproval  6m57s  cert-manager-certificaterequests-issuer-venafi      Not signing CertificateRequest until it is Approved
+  Normal  WaitingForApproval  6m57s  cert-manager-certificaterequests-issuer-acme        Not signing CertificateRequest until it is Approved
+  Normal  WaitingForApproval  6m57s  cert-manager-certificaterequests-issuer-ca          Not signing CertificateRequest until it is Approved
+  Normal  cert-manager.io     6m57s  cert-manager-certificaterequests-approver           Certificate request has been approved by cert-manager.io
+  Normal  OrderCreated        6m57s  cert-manager-certificaterequests-issuer-acme        Created Order resource aks-istio-ingress/bookinfo-letsencrypt-certs-1-507153367
+  ```
+
+4. check order resource.
+```shell
+kubectl describe order bookinfo-letsencrypt-certs-1-507153367 -n aks-istio-ingress
+```
+```console
+Name:         bookinfo-letsencrypt-certs-1-507153367
+Namespace:    aks-istio-ingress
+Labels:       <none>
+Annotations:  cert-manager.io/certificate-name: bookinfo-letsencrypt-certs
+              cert-manager.io/certificate-revision: 1
+              cert-manager.io/private-key-secret-name: bookinfo-letsencrypt-certs-hpjst
+...
+Events:
+  Type    Reason   Age   From                 Message
+  ----    ------   ----  ----                 -------
+  Normal  Created  11m   cert-manager-orders  Created Challenge resource "bookinfo-letsencrypt-certs-1-507153367-692923294" for domain "test.dev.azureservicemesh.io"
+```
+
+5. check challenge resource.
```

Comment:
what does the user need to check inside this resource?

Reply:
Just wanted to let the users know that these are the chain of resources provisioned by cert-manager and they should troubleshoot these in case something doesn't work for them. Added the header to check for readiness of these resources.